integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/DomainUtils.java

```diff
@@ -991,6 +993,175 @@ public static DomainResource createDomainInImageAndVerify(String domainUid,
     return domain;
   }
 
+
+  /**
+   * Create domain resource with configMap.
+   *
+   * @param domainUid unique id of the WebLogic domain
+   * @param domainNamespace domain namespace
+   * @param clusterName names of cluster
+   * @param miiImage name of the image including its tag
+   * @param wlSecretName wls admin secret name
+   * @param repoSecretName name of the secret for pulling the WebLogic image
+   * @param encryptionSecretName name of the secret used to encrypt the models
+   * @param replicaCount replica count of the cluster
+   * @param configmapName name of the configMap containing WebLogic Deploy Tooling model
+   * @param introspectorDeadline seconds of introspector job active deadline
+   * @param failureRetryLimitMinutesArgs the time in minutes before the operator will stop retrying Severe failures
+   * @return oracle.weblogic.domain.Domain object
+   */
+  public static  DomainResource createDomainResourceWithConfigMap(String domainUid,
```

Comment:
Can you change the method name to createMiiDomainResourceWithConfigMap(...) since it is for model-in-image domain? also update the javadoc.

Reply:
changed